packages/origin-backend-app/src/main.ts

```diff
@@ -1,5 +1,5 @@
 import { AppModule as ExchangeModule } from '@energyweb/exchange';
-import { LoggerService } from '@nestjs/common';
+import { LoggerService, ValidationPipe } from '@nestjs/common';
```

Comment:
Are these change necessary? ValidationPipe is already set a provider

https://github.com/energywebfoundation/origin/blob/376b6ba4b3e7487b731e5bcbd9d27d81a4ba4580/packages/exchange/src/app.module.ts#L67